### DISCUSSION TITLE
rendering osgearth in osgQt

### DISCUSSION THREAD

**Discussion by FuXiii** [pelicanmapping/osgearth#2186] [Support Q&A]
I try to rending osgearth in [osgQt](https://github.com/mathieu/osgQt)

the code very simple:
![image](https://user-images.githubusercontent.com/40132375/204241176-7242402e-df7d-4ab9-8cc6-3293545f3d6a.png)

and the render result:
![image](https://user-images.githubusercontent.com/40132375/204241393-604cc18b-e263-4a65-9391-31bd0b83f678.png)

the eath become transparent. How to fix it？

**Comment by gwaldron** [comment]
This may be of help:
https://docs.osgearth.org/en/latest/faq.html#rendering-problems-when-using-the-qt-qopenglwidget
